src/main/java/baseball/model/AnswerChecker.java

```diff
@@ -0,0 +1,46 @@
+package baseball.model;
+
+public class AnswerChecker {
+    private int ball = 0;
+    private int strike = 0;
+    private boolean isAnswer = false;
+
+    public void checkAnswer (String answer, String input) {
+        predictionValidator(answer, input);
+        reset();
+        for (int i = 0; i < answer.length(); i++) {
+            if (answer.charAt(i) == input.charAt(i)) {
+                strike++;
+            } else if (answer.contains(Character.toString(input.charAt(i)))){
+                ball++;
+            }
+        }
+        isAnswer = strike == answer.length() && ball == 0;
+    }
+
+    private void predictionValidator(String answer, String input) {
```

Comment:
```suggestion
    private void validatePrediction(String answer, String input) {
```
메서드명은 동사, 변수명은 명사형으로 처리하면 조금 더 직관적으로 읽히는 경우가 많은 것 같아서, 위 예시처럼 다른 이름을 찾아도 좋을 것 같습니다!